### ISSUE TITLE
Torch 2.0 not compatible

### ISSUE THREAD

**Issue by vic-t** [Amblyopius/Stable-Diffusion-ONNX-FP16#29]
I couldn't get this running with Torch 2.0. So, I reinstalled Torch 1.13.1 and then it worked. Maybe the requirements file should be changed accordingly.

**Comment by vic-t** [comment]
To confirm the error, I upgraded to Torch 2.0 again and reran the test-txt2img again. That worked fine. I also reran conv_sd_to_onnx. That one failed with the following message: 

 ```
ERROR: missing-standard-symbolic-function
=========================================
Exporting the operator 'aten::scaled_dot_product_attention' to ONNX opset version 15 is not supported. Please feel free to request support or submit a pull request on PyTorch GitHub: https://github.com/pytorch/pytorch/issues.
None
<Set verbose=True to see more details>


Traceback (most recent call last):
  File "D:\ai\amd310\Stable-Diffusion-ONNX-FP16\conv_sd_to_onnx.py", line 654, in <module>
    convert_models(pl, args.output_path,
  File "D:\ai\amd310\lib\site-packages\torch\utils\_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "D:\ai\amd310\Stable-Diffusion-ONNX-FP16\conv_sd_to_onnx.py", line 373, in convert_models
    onnx_export(
  File "D:\ai\amd310\Stable-Diffusion-ONNX-FP16\conv_sd_to_onnx.py", line 141, in onnx_export
    export(
  File "D:\ai\amd310\lib\site-packages\torch\onnx\utils.py", line 506, in export
    _export(
  File "D:\ai\amd310\lib\site-packages\torch\onnx\utils.py", line 1548, in _export
    graph, params_dict, torch_out = _model_to_graph(
  File "D:\ai\amd310\lib\site-packages\torch\onnx\utils.py", line 1117, in _model_to_graph
    graph = _optimize_graph(
  File "D:\ai\amd310\lib\site-packages\torch\onnx\utils.py", line 665, in _optimize_graph
    graph = _C._jit_pass_onnx(graph, operator_export_type)
  File "D:\ai\amd310\lib\site-packages\torch\onnx\utils.py", line 1901, in _run_symbolic_function
    raise errors.UnsupportedOperatorError(
torch.onnx.errors.UnsupportedOperatorError: Exporting the operator 'aten::scaled_dot_product_attention' to ONNX opset version 15 is not supported. Please feel free to request support or submit a pull request on PyTorch GitHub: https://github.com/pytorch/pytorch/issues.
```

That error does not appear with Torch 1.13.1

**Comment by Amblyopius** [comment]
So apparently diffusers devs have decided to use a torch 2.0 specific attention processor for VAE without the option to turn it off. In prior diffusers versions they enabled it for UNET only and offered an option to turn it off (which we did as part of conversion).
Bad news: We'll have to switch to using nightly torch 2.1 builds (until 2.1 is released) as the export of scaled_dot_product_attention is fixed in 2.1
Good news: The implementation of scaled_dot_product_attention provides a speed boost

I'll change the instructions and we'll install 2.1 nightly builds of torch instead of main line
I'll also remove the attention processor switching from the conversion script and do some testing (I've got no idea yet whether/how it impacts VRAM use)

**Comment by Amblyopius** [comment]
Fixed by moving to torch 2.1 (nightly)
